test/ngAnimate/animateSpec.js

```diff
@@ -459,6 +459,26 @@ describe('animations', function() {
       expect(commentNode[0].parentNode).not.toBe(parentNode);
     }));
 
+    it('enter() should animate a transcluded clone with `templateUrl`', function() {
+      module(function($compileProvider) {
+        $compileProvider.directive('foo', valueFn({templateUrl: 'foo.html'}));
+      });
+
+      inject(function($animate, $compile, $rootScope, $templateCache) {
+        parent.append(jqLite('<foo ng-if="showFoo"></foo>'));
+        $templateCache.put('foo.html', '<div>FOO</div>');
+        $compile(parent)($rootScope);
+
+        expect(capturedAnimation).toBe(null);
```

Comment:
Ah, it's `undefined` only in one test:
https://github.com/gkalpak/angular.js/blob/acd9e7b757cdb9f04ae7a12ff21fdc8d27f314b5/test/ngAnimate/animateSpec.js#L21-L46

I guess the changes are fine then.